web/package.json

```diff
@@ -106,6 +110,7 @@
     ],
     "rules": {
       "react/jsx-props-no-spreading": 0,
+      "react/destructuring-assignment": 0,
```

Comment:
@tobiaslins don't you like this rule?

Reply:
It's not possible to use `props.ariaLabel` for example. This is needed in order to use type guards and correct typings